src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbErrorCodeMessages.cs

```diff
@@ -0,0 +1,755 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FirebirdSql.Data.FirebirdClient;
+/// <summary>
+/// Firebird Error messages.
+/// 
+/// Messages are taken from documentation at: https://ib-aid.com/download/docs/firebird-language-reference-2.5/fblangref25-appx02-sqlcodes.html
+/// </summary>
+public partial class FbErrorCodes
+{
+	public static string GetMessage(int code) =>
```

Comment:
These messages are already part of exception message (from `IscErrorMessages.cs`).